packages/nx/src/command-line/nx-commands.ts

```diff
@@ -1022,14 +1023,14 @@ function runMigration() {
   }
 }
 
-function nxCliPath() {
+function nxCliPath(
+  version = process.env.NX_MIGRATE_USE_NEXT === 'true' ? 'next' : 'latest'
+) {
```

Comment:
This is a weird function name, can we change it to like... `installNxTemporarily(version: 'latest' | 'next')`?

Don't make it optional, and move the environment variable check outside of here.